invoice-app/src/Container/CreateBill/index.js

```diff
@@ -0,0 +1,338 @@
+                                        // })}
+                                        value={billFrom?.vendorName}
+                                    />
+
+                                    <TextField name="vendorEmail" fullWidth label="Email Address"
+                                        {...register('vendorEmail', {
+                                            required: errorMsg,
+                                        })}
+                                        margin="dense"
+                                        size="small"
+                                        id="fullWidth"
+                                        error={errors.vendorEmail}
+                                        helperText={errors.vendorEmail?.message}
+                                    // onChange={data => setBillFrom({
+                                    //     vendorEmail: data.vendorEmail
+                                    // })}
+                                    // value={billFrom.vendorEmail}
+                                    />
+
+                                    <TextField name="vendorContact" fullWidth label="Contact"
+                                        {...register('vendorContact', {
+                                            required: errorMsg,
+                                        })}
+                                        margin="dense"
+                                        size="small"
+                                        id="fullWidth"
+                                        error={errors.vendorContact}
+                                        helperText={errors.vendorContact?.message}
+                                    // onChange={data => setBillFrom({
+                                    //     vendorContact: data.vendorContact
+                                    // })}
+                                    // value={billFrom.vendorContact}
+                                    />
+
+                                    <TextField name="vendorAddress" fullWidth label="Address"
+                                        {...register('vendorAddress', {
+                                            required: errorMsg,
+                                        })}
+                                        margin="dense"
+                                        size="small"
+                                        id="fullWidth"
+                                        error={errors.vendorAddress}
+                                        helperText={errors.vendorAddress?.message}
+                                    // onChange={data => setBillFrom({
+                                    //     vendorAddress: data.vendorAddress
+                                    // })}
+                                    // value={billFrom.vendorAddress}
+                                    />
+
+                                    <TextField name="vendorCityPin" fullWidth label="City and Pin-code"
```

Comment:
use separate fields for city and pincode. Add validation for pincode(like minimum 5 digits and max 7 digits as LIKE indian standard pincode)